lib/messages/commands/version.js

```diff
@@ -22,6 +22,8 @@ var packageInfo = require('../../../package.json');
  * @param {BN=} arg.services
  * @param {Date=} arg.timestamp
  * @param {Number=} arg.startHeight
+ * @param {boolean} [arg.relay]
+ * @param {string} [arg.mnauthChallenge]
```

Comment:
This name is using somewhere else? Maybe we rename it to `mnAuthChallenge`?

Reply:
ok, looks better somehow 👍 
Updated!